src/renderer/components/dialogs/CreateChat.tsx

```diff
@@ -283,69 +283,68 @@ export function useGroupMembers(initialMemebers: number[]) {
   ]
 }
 
-export const GroupSettingsSetNameAndProfileImage = ({
+export const ChatSettingsSetNameAndProfileImage = ({
   groupImage,
   onSetGroupImage,
   onUnsetGroupImage,
-  groupName,
-  setGroupName,
-  errorMissingGroupName,
-  setErrorMissingGroupName,
+  chatName,
+  setChatName,
+  errorMissingChatName,
+  setErrorMissingChatName,
   color,
+  type,
 }: {
   groupImage?: string | null
-  onSetGroupImage: () => void
-  onUnsetGroupImage: () => void
-  groupName: string
-  setGroupName: (newGroupName: string) => void
-  errorMissingGroupName: boolean
-  setErrorMissingGroupName: React.Dispatch<React.SetStateAction<boolean>>
+  onSetGroupImage?: () => void
+  onUnsetGroupImage?: () => void
+  chatName: string
+  setChatName: (newGroupName: string) => void
+  errorMissingChatName: boolean
+  setErrorMissingChatName: React.Dispatch<React.SetStateAction<boolean>>
   color?: string
+  type: 'group' | 'broadcast'
 }) => {
   const tx = useTranslationFunction()
   const onChange = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
-    if (target.value.length > 0) setErrorMissingGroupName(false)
-    setGroupName(target.value)
+    if (target.value.length > 0) setErrorMissingChatName(false)
+    setChatName(target.value)
+  }
+  if (type === 'group' && !(onSetGroupImage && onUnsetGroupImage)) {
+    throw new Error(
+      'if type is group, onSetGroupImage and onUnsetGroupImage must be present'
+    )
   }
   return (
     <>
       <div className='group-settings-container'>
-        <GroupImage
-          style={{ float: 'left' }}
-          groupImage={groupImage}
-          onSetGroupImage={onSetGroupImage}
-          onUnsetGroupImage={onUnsetGroupImage}
-          groupName={groupName}
-          color={color}
-        />
-        <input
-          className='group-name-input'
-          placeholder={tx('group_name')}
-          value={groupName}
-          onChange={onChange}
-          autoFocus
-          style={{
-            marginLeft: '17px',
-            width: 'calc(100% - 65px - 17px)',
-            top: '-7px',
-            position: 'relative',
-          }}
-          spellCheck={false}
-        />
+        {type === 'group' && onUnsetGroupImage && onSetGroupImage && (
+          <GroupImage
+            style={{ float: 'left' }}
+            groupImage={groupImage}
+            onSetGroupImage={onSetGroupImage}
+            onUnsetGroupImage={onUnsetGroupImage}
+            groupName={chatName}
+            color={color}
+          />
+        )}
+        <div className='group-name-input-wrapper'>
+          <input
+            className='group-name-input'
+            placeholder={type === 'group' ? tx('group_name') : tx('group_name')}
```

Comment:
here also other string for broadcast list is needed

Reply:
yeah